include/RAJA/pattern/forall.hpp

```diff
@@ -465,7 +481,18 @@ RAJA_INLINE void forall(Args&&... args)
 
   util::callPostLaunchPlugins(context);
 }
+template <typename ExecutionPolicy, typename... Args>
+RAJA_INLINE RAJA::resources::Event forall(RAJA::resources::Resource *r, Args&&... args)
```

Comment:
Why is it a resource pointer and not a reference?

Reply:
No reason, this is fixed now.